srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Manager/Soap/AcoreSoap.php

```diff
@@ -18,7 +18,7 @@ public function isConfigured()
         return $this->params != null;
     }
 
-    public function executeCommand($command)
+    public function executeCommand($command, $logCommand = false, $orderId = null)
```

Comment:
I would put this as `true` by default, but if there is a strong reason for it, we can leave as it is

Reply:
I put as false because I do not recommend it for account related commands, do to avoid expose passwords and other stuff related. But I could change it to true and for account commands add the false.

Reply:
mhm right, for the account password can be an issue, so that's ok